tests/resources/setup.sh

```diff
@@ -1,36 +1,22 @@
 #!/bin/bash
 
-usage() {
-    echo "Usage: setup.sh <flavor>"
-    echo "Flavor can be 'standard', 'noadmin', 'noauth', 'taskadmins'"
-}
+set -e
 
-if (( $# != 1 )); then
-    echo "Illegal number of parameters"
-    usage
-    exit 1
-fi
+# just in case showing docker status
+docker ps -a | head -n10
 
-source bootstrap.sh
-
-docker-compose -f docker-compose.yml -f tests/$1/docker-compose.yml build
-docker-compose -f docker-compose.yml -f tests/$1/docker-compose.yml up -d zookeeper mesos-slave mesos-master marathon openldap
-docker ps -a
-
-
-echo "Wait for LDAP to be ready"
+echo "TODO: Wait for LDAP to be ready"
 sleep 5
 
-# import users in LDAP
-docker run -it --network host -v $(pwd)/tests/resources/ldap/base.ldif:/base.ldif --rm mbentley/ldap-utils ldapadd -h 172.16.130.6 -D "cn=admin,dc=example,dc=com" -f /base.ldif -w password -x
-docker-compose -f docker-compose.yml -f tests/$1/docker-compose.yml up -d mesos-term mesos-term-ui
+# import users in LDAP. This operation is not idempotent and will fail if users already exists
+# to reset: sudo docker-compose stop openldap; sudo docker-compose rm openldap; sudo docker-compose up -d openldap
+docker run -t --network host -v $(pwd)/tests/resources/ldap/base.ldif:/base.ldif --rm mbentley/ldap-utils ldapadd -h 172.16.130.6 -D "cn=admin,dc=example,dc=com" -f /base.ldif -w password -x
```

Comment:
FAR: ldapModify command may allow you to achieve that (but it would need a rewrite of ldif file I think)